README.md

```diff
@@ -33,62 +33,65 @@ Before proceeding with the setup, ensure that the following software are install
-    
-    - Ensure that the following directory available in the /zephyr directory:
-        - /soc_debugger -> OpenOCD Configuration script for Efinix Sapphire SoC: [github:sapphire-soc-debugger](https://github.com/Efinix-Inc/sapphire-soc-debugger)
-        - /dt-generator -> Device Tree Generator repository for Efinix Saphhire SoC [github:sapphire-soc-dt-generator](https://github.com/Efinix-Inc/sapphire-soc-dt-generator)
-        - /zephyr       -> Zephyr repository for Zephyr project [github:zephyr](https://github.com/Efinix-Inc/zephyr)
-    > ***Note:*** If any of the directory is not exist in the /zephyr directory, you may manually clone the respective repository. 
-    - Open the terminal in VS Code, In the command palette, type `Terminal: Create New Terminal (In Active Workspace)` and hit enter
-    
-    - To use your own SoC Configuration, follow the steps below; 
-        - Generate your own SoC with our Efinity IP Manager
-        - Locate the soc.h which will be available in `embedded_sw/<soc name>/bsp/efinix/EfxSapphireSoc/include/soc.h`
-        - Copy the soc.h from your SoC folder into the dt-generator folder
-        - The dt-generator will help you to update all the necessary files with your SoC configuration
-        - Run the following command: 
+
+   - Ensure that the following directory available in the /zephyr directory: - /soc_debugger -> OpenOCD Configuration script for Efinix Sapphire SoC: [github:sapphire-soc-debugger](https://github.com/Efinix-Inc/sapphire-soc-debugger) - /dt-generator -> Device Tree Generator repository for Efinix Saphhire SoC [github:sapphire-soc-dt-generator](https://github.com/Efinix-Inc/sapphire-soc-dt-generator) - /zephyr -> Zephyr repository for Zephyr project [github:zephyr](https://github.com/Efinix-Inc/zephyr)
+     > **_Note:_** If any of the directory is not exist in the /zephyr directory, you may manually clone the respective repository.
+   - Open the terminal in VS Code, In the command palette, type `Terminal: Create New Terminal (In Active Workspace)` and hit enter
+
+   - In order for west to fetch all the necessary modules and files, west needs to know a specified workspace.
+     - Run `west init` in zephyr (outside of the zephyr os source directory).
+     - Run `west update` to download some additional stuff.
+     - This will initialize the workspace so that the `west build` command can be executed.
+
+   - To use your own SoC Configuration, follow the steps below;
+     - Generate your own SoC with our Efinity IP Manager
+     - Locate the soc.h which will be available in `embedded_sw/<soc name>/bsp/efinix/EfxSapphireSoc/include/soc.h`
+     - Copy the soc.h from your SoC folder into the dt-generator folder
+     - The dt-generator will help you to update all the necessary files with your SoC configuration
+     - Run the following command:
+
 ```
 python3 zephyr_installer.py <soc name> <board name> <targeted device> ./zephyr/ ./dt-generator/soc.h <-m memory configuration>
```

Comment:
Can you help to update from `zephyr_installer.py` to `dt-generator/zephyr_installer.py`